source/module_hamilt/ks_lcao/deepks_lcao.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __DEEPKSLCAO
+#define __DEEPKSLCAO
```

Comment:
```suggestion
#define DEEPKSLCAO
```

To be consistent with other ABACUS header files, I think it is better to remove the "__" prefix